packages/accounts-controller/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Bump `@metamask/utils` from `^11.8.0` to `^11.8.1` ([#6708](https://github.com/MetaMask/core/pull/6708))
 - Bump `@metamask/utils` from `^11.4.2` to `^11.8.0` ([#6588](https://github.com/MetaMask/core/pull/6588))
```

Comment:
Can you combine these? From the perspective of the current version, we just know that `utils` is ^11.4.2 now and will be ^11.8.1 after this new version is published, we don't need to keep track of the intermediate steps.

``` suggestion
- Bump `@metamask/utils` from `^11.4.2` to `^11.8.1` ([#6588](https://github.com/MetaMask/core/pull/6588), [#6708](https://github.com/MetaMask/core/pull/6708))
```

There are other changelogs where we can combine this as well, can you take a look at those?